### PR TITLE
[GITHUB-1] Adds New Relic role

### PR DESCRIPTION
**Pull request by lobsterdore**

### CODE REVIEW

**Comment by sreddel** [comment]
re the commit message and PR title.. whats a "rolic"? :D

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## IDEs, editors, ...
+.DS_Store
+.idea
+.project
+*.bak
+*.iml
+*.sublime*
+*.swp
+/.settings
+
```
Comment (dennisconrad) [inline .gitignore:10]: I don't know why we are having multiple newlines here, but at this stage I'm afraid to ask.

File: tasks/configure.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    dest: "{{ sansible_newrelic.integrations.php.ini_paths.base }}"
+    mode: 0640
+  when: sansible_newrelic.enabled
+    and sansible_newrelic.integrations.php.enabled
+
+- name: Enable PHP integration
+  become: yes
+  file:
+    dest: "{{ item }}"
+    mode: 731
```
Comment (sreddel) [inline tasks/configure.yml]: Im not sure it makes sense to specify a mode with a symlink?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  only:
+    - develop
+    - master
+
+env:
+  - ANSIBLE_INSTALL_VERSION=2.0.2.0
+  - ANSIBLE_INSTALL_VERSION=2.1.6.0
+  - ANSIBLE_INSTALL_VERSION=2.2.3.0
+  - ANSIBLE_INSTALL_VERSION=2.3.2.0
+  - ANSIBLE_INSTALL_VERSION=2.4.1.0
```
Comment (dennisconrad) [inline .travis.yml]: Ansible 2.3.3 and 2.4.2.0 are out.

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016
```
Comment (dennisconrad) [inline LICENSE]: 2017?  I think we should update scaffold.

File: README.md [hunk 5]
```diff
@@ -1 +1,101 @@
...
+
+
+
+
+## Tags
+
+This role uses tags: **build** and **configure**
+
+* `build` - Installs ...
+* `configure` - Configures ...
```
Comment (dennisconrad) [inline README.md]: Are those three dots deliberate?

File: .gitignore (hunk 1 above)
Comment (sreddel) [reply .gitignore:10]: Book of scaffold. Chapter 1, verse 10

File: tasks/configure.yml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  when: sansible_newrelic.enabled
+
+- name: Configure PHP integration
+  become: yes
+  template:
+    src: integrations/php/newrelic.ini.j2
+    dest: "{{ sansible_newrelic.integrations.php.ini_paths.base }}"
+    mode: 0640
+  when: sansible_newrelic.enabled
+    and sansible_newrelic.integrations.php.enabled
```
Comment (dennisconrad) [inline tasks/configure.yml]: Might be easier to read as

```yaml
when:
  - sansible_newrelic.enabled
  - sansible_newrelic.integrations.php.enabled
```

(see line 30 below as well)

File: .gitignore (hunk 1 above)
Comment (lobsterdore) [reply .gitignore:10]: This is a Wojtek thing, he loved four lines between everything because of some blog post.

File: .travis.yml (hunk 3 above)
Comment (lobsterdore) [reply .travis.yml]: They aren't available via Pip yet.

File: LICENSE (hunk 4 above)
Comment (lobsterdore) [reply LICENSE]: It's a time travelling role! I will update here and the scaffold.

File: README.md (hunk 5 above)
Comment (lobsterdore) [reply README.md]: D'oh, need to fill in the blanks.

**Review by sreddel — APPROVED** [review]
LGTM

**Pull request merged**